vscode/microsoft-kiota/src/kiotaInterop.ts

```diff
@@ -197,30 +197,30 @@ export function maturityLevelToString(level: MaturityLevel): string {
     }
 }
 export enum DependencyType {
-  abstractions,
-  serialization,
-  authentication,
-  http,
-  bundle,
-  additional,
+    abstractions,
+    serialization,
+    authentication,
+    http,
+    bundle,
+    additional,
 }
 export function dependencyTypeToString(type: DependencyType): string {
-  switch (type) {
-    case DependencyType.abstractions:
-      return "abstractions";
-    case DependencyType.serialization:
-      return "serialization";
-    case DependencyType.authentication:
-      return "authentication";
-    case DependencyType.http:
-      return "http";
-    case DependencyType.bundle:
-      return "bundle";
-    case DependencyType.additional:
-      return "additional";
-    default:
-      throw new Error("unknown type");
-  }
+    switch (type) {
+        case DependencyType.abstractions:
+            return "abstractions";
+        case DependencyType.serialization:
+            return "serialization";
+        case DependencyType.authentication:
+            return "authentication";
+        case DependencyType.http:
+            return "http";
+        case DependencyType.bundle:
+            return "bundle";
+        case DependencyType.additional:
+            return "additional";
+        default:
+            throw new Error("unknown type");
+    }
```

Comment:
If we used TypeScript's [Reverse mappings](https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings) this code could be shortened.

```suggestion
    const stringType = DependencyType[type];
    if (stringType) {
        return stringType;
    }
    throw new Error("unknown type");
```

Reply:
It's likely possible. However the code changes are not in the scope of this PR. I believe a diff was created because of the whitespace formatting.